### PR TITLE
Fixes #197 difference qualification plan

### LINKED ISSUE #197: Fields in qualification plan not appearing in configuration plan
[Qualification plan](https://github.com/Open-Systems-Pharmacology/QualificationPlan/files/8482516/remote_master_quali_plan.zip) from https://github.com/Open-Systems-Pharmacology/QualificationPlan/blob/main/examples/remote/input/master_quali_plan.json

[Resulting configuration plan from qualification runner ](https://github.com/Open-Systems-Pharmacology/QualificationPlan/files/8482538/remote-report-configuration-plan.zip)

Comparison of the two files:  

image

1 - There is no "Reference" field in the configuration plan.
2 - The configuration plan still has an ID field when there isn't one in the qualification plan.

### PR DESCRIPTION
**Pull request by msevestre**

### CODE REVIEW

**Comment by msevestre** [comment]
@abdullahhamadeh @Yuri05 Generated everything again. That should be good now

File: examples/remote/reporting engine input/Inputs/Midazolam/Compound/Midazolam.md [hunk 1]
```diff
@@ -1,43 +1,43 @@
...
-## Calculation methods
-
-Name                    | Value               |
------------------------ | ------------------- |
-Partition coefficients  | Rodgers and Rowland |
-Cellular permeabilities | PK-Sim Standard     |
-## Processes
-
-### Metabolizing Enzyme: CYP3A4-Optimization
+## Compound: Midazolam
```
Comment (msevestre) [inline examples/remote/reporting engine input/Inputs/Midazolam/Compound/Midazolam.md:1]: @Yuri05 is this a bug here? Or wasn't it generated properly before?

File: examples/remote/reporting engine input/Inputs/Midazolam/Simulation/S1.md [hunk 2]
```diff
@@ -1,15 +1,2 @@
-# Simulation: S1
+## Simulation: S1
 
-Simulation 'S1'
```
Comment (msevestre) [inline examples/remote/reporting engine input/Inputs/Midazolam/Simulation/S1.md:3]: @Yuri05 Something is wrong here in the way the Marldown is generated. None of the info is there anymore. THis is an issue in PKSim. I think the identation is right however

**Comment by abdullahhamadeh** [comment]
> @abdullahhamadeh @Yuri05 Generated everything again. That should be good now

Thanks @msevestre .  What do I need to update on my machine?

**Comment by msevestre** [comment]
@abdullahhamadeh just take the code from the branch where the PR is coming from

**Pull request merged**